src/change-listeners.js

```diff
@@ -269,6 +276,9 @@ class ChangeListeners extends Process {
     try {
       await this._processBatchOfChanges(listener)
     } catch (err) {
+      if (err instanceof DatabaseNotFoundError) {
```

Comment:
I think we probably want this if block below the `this._onError(err)` line so that the error is logged, etc...

Reply:
Oh, nice catch! No, this looks good. I'll merge this now